source/common/runtime/runtime_features.cc

```diff
@@ -116,6 +116,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_refresh_rtt_after_request);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_reject_all);
 // TODO(adisuissa): enable by default once this is tested in prod.
 FALSE_RUNTIME_GUARD(envoy_restart_features_use_eds_cache_for_ads);
+// TODO(wbpcode): enable by default after a complete deprecation period.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_no_downgrade_to_canonical_name);
```

Comment:
I don't think we have the concept of a deprecation period for flipping things true.  how about altering this to flip true after the Q4 release, which gives folks a couple of months to complain about the behavior at HEAD?

Reply:
When I say deprecation period, I mean 6 months.
>  how about altering this to flip true after the Q4 release, which gives folks a couple of months to complain about the behavior at HEAD?
sgtm.